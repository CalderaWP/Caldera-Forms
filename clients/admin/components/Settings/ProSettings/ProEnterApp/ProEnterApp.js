import React from 'react';
import classNames from 'classnames'

/**
 * Create the global form settings UI
 * @param {Object} props
 * @return {*}
 * @constructor
 */
export const ProEnterApp = (props) => {
	return(
		<div
			className={classNames(props.className,ProEnterApp.classNames.wrapper)}
		>
			ProEnterApp
		</div>
	)
};



/**
 * Class names used in the GlobalForms settings component
 * @type {{wrapper: string}}
 */
ProEnterApp.classNames = {
	wrapper: 'caldera-forms-global-form-settings'
};