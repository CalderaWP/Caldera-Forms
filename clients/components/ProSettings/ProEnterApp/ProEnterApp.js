import React from 'react';
import classNames from 'classnames'

/**
 * Create the global form settings UI
 * @param {Object} props
 * @return {*}
 * @constructor
 */
export const ProEnterApp = (props) => {
	return(
		<div
			className={classNames(props.className,ProEnterApp.classNames.wrapper)}
		>
			<a href={'/wp-admin/admin.php?page=cf-pro'} >Go to Pro Settings</a>
		</div>
	)
};



/**
 * Class names used in the GlobalForms settings component
 * @type {{wrapper: string}}
 */
ProEnterApp.classNames = {
	wrapper: 'caldera-forms-global-form-settings'
};