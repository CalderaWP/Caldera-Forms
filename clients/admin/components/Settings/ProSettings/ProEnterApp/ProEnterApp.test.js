import renderer from 'react-test-renderer';
import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {ProEnterApp} from "./ProEnterApp";

Enzyme.configure({ adapter: new Adapter() });

describe( 'ProEnterApp component', () => {
	it( 'Matches snapshot with minimal props', () => {
		expect(
			renderer.create(
				<ProEnterApp/>
			).toJSON()
		).toMatchSnapshot()
	});

	it( 'Is wrapped in the right class', () => {
		expect(
			shallow(
				<ProEnterApp/>
			).find( '.' + ProEnterApp.classNames.wrapper )
				.length
		).toEqual(1)
	});
});