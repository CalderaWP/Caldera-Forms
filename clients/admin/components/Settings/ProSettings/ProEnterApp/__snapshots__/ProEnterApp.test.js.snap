// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProEnterApp component Matches snapshot with minimal props 1`] = `
<div
  className="caldera-forms-global-form-settings"
>
  ProEnterApp
</div>
`;
